libshvvisu/src/timeline/channelprobewidget.cpp

```diff
@@ -21,7 +21,16 @@ ChannelProbeWidget::ChannelProbeWidget(ChannelProbe *probe, QWidget *parent) :
 	ui->edCurentTime->setStyleSheet("background-color: white");
 	ui->fHeader->setStyleSheet("background-color:" + m_probe->color().name() + ";");
 	ui->tbClose->setStyleSheet("background-color:white;");
-	ui->lblTitle->setStyleSheet("color:white;");
+	QColor c = m_probe->color();
+#if QT_VERSION_MAJOR >= 6
+	float a = 1 - ( 0.299F * c.redF() + 0.587F * c.greenF() + 0.114F * c.blueF());
+	if (c.alphaF() > 0.0F && a >= 0.3F) {
+#else
+	double a = 1 - ( 0.299 * c.redF() + 0.587 * c.greenF() + 0.114 * c.blueF());
+	if (c.alphaF() > 0.0 && a >= 0.3) {
```

Comment:
try to use L channel of `QColor::hsl()`

Reply:
Opraveno, nakonec jsem nasel jeste funkci `int QColor::lightness()`, ktera vraci presne to co potrebujeme.